### PR TITLE
Consolidation of our work

### PR DESCRIPTION
**Pull request by ajsun**
## Summary of Changes
PLEASE BUILD OFF OF THIS BRANCH. 
1. folder structure in the components file has been flattened a bit. Screens + Navigators now sit at the top level of the src folder
2. Login flow designs have been improved - login now works, as well as sign up flow. its currently hooked up with staging
3. navigator names are cleaned a bit

TODO

1. we need to think through the foldered components, i dont like them
2. consolidating common styles into a single index.js
3. outline of flows for the navigator + best organization structure

### CODE REVIEW

**Comment by ajsun** [comment]
👻 This branch as been deployed to:
            (https://expo.io/@poppop/nutmeg?release-channel=as-consolidating)

File: src/screens/index.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
-import HomeScreen from './HomeScreen.js';
+import HomeScreen from './HomeScreen';
 import DetailsScreen from './DetailsScreen';
-import LoginScreen from './LoginScreen';
-import SignupScreen from './SignupScreen';
+import SignUpScreen from './SignUpScreen';
```
Comment (jonathanandrewsuk) [inline src/screens/index.js]: The file is called "SignupScreen", change the file name to a capital "U"

File: src/styles/common.js [hunk 2]
```diff
@@ -10,7 +10,17 @@ export const COLOR_TERTIARY = '#569BD4'; // Light Blue
...
 export const FONT_NORMAL = 'OpenSans-Regular';
 export const FONT_BOLD = 'OpenSans-Bold';
+
+// SIZES
+
+export const SIZE_XSMALL = 8;
+export const SIZE_SMALL = 12;
+export const SIZE_MEDIUM = 16;
+export const SIZE_LARGE = 24;
+export const SIZE_XLARGE = 32;
```
Comment (jonathanandrewsuk) [inline src/styles/common.js]: Numerical font size is fine for now, but I'd like to implement something more dynamic based on screen width soon.

File: src/screens/index.js (hunk 1 above)
Comment (ajsun) [reply src/screens/index.js]: i did, you may have to set your git config to be case sensitive `git config core.ignorecase false`

**Comment by jonathanandrewsuk** [comment]
forgot the 🍏!

**Pull request merged**